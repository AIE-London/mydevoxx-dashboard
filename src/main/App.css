.App {
  text-align: left;
  background-color: dimgrey;
  color: white;
}

.App-day {
   font-size: xx-large;
   text-indent: 50px;
   color: orange;
   margin: 5px 0;
 }

.App-time {
  font-size: large;
  text-indent: 50px;
  font-weight: 100;

}


