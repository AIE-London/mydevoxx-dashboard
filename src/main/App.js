import React, { Component } from 'react';
import TalkCard from './components/TalkCard/';
import './App.css';
import {Col, Row} from 'react-flexbox-grid';

class App extends Component {
  render() {
    return (
      <div className="App">
        <div style={{height: '72px'}}>
        </div>
        <Row center="xs">
          <Col xs={10}>
            <TalkCard title="Welcome to Devoxx 2017"></TalkCard>
          </Col>
        </Row>
      </div>
    );
  }
}

export default App;
