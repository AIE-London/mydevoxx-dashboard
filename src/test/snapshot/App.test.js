import React from 'react';
import ReactDOM from 'react-dom';
import App from '../../main/App';
import renderer from 'react-test-renderer';


let reportStatsData =
    {
        "minutes": 455,
        "talks": 10,
        "learning": "JS, Polymer, Java" ,
        "attendees": 435
    };

test('app component', () => {

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<App />, div);
  });

});


test('app component snapshot', () => {
    const tree = renderer.create(
      <App reportStats={reportStatsData}/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
});