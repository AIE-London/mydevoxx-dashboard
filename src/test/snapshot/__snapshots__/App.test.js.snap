exports[`test app component snapshot 1`] = `
<div
  className="App">
  <div
    style={
      Object {
        "height": "72px",
      }
    } />
  <div
    className="row center-xs">
    <div
      className="col-xs-10">
      <div
        className="bvnlUe row"
        left="xs">
        <div
          className="col-xs-6 fteZLe">
          <div
            className="row"
            left="xs">
            <h2
              className="dwcXfU"
              id="title">
              Welcome to Devoxx 2017
            </h2>
          </div>
          <p
            className="App-intro">
            To get started, edit 
            <code>
              src/App.js
            </code>
             and save to reload.
          </p>
        </div>
        <div
          className="col-xs-6 eZvXOT">
          <div
            className="row"
            left="xs">
            <h3
              className="jJURiy"
              id="notesHeader">
              My Notes
            </h3>
          </div>
          <div
            className="row"
            left="xs">
            <p
              className="cDpRHH">
              Lorem ipsum dolor sit amet, everti quaestio mel ea. Ex eos volutpat qualisque. Sale tantas cotidieque quo ut, ad nostro consectetuer nec. Feugiat qualisque quo an. Labores officiis te nam.
            </p>
          </div>
          <div
            className="row"
            left="xs">
            <h3
              className="jJURiy"
              id="notesHeader">
              My Reviews
            </h3>
          </div>
          <div
            className="row"
            left="xs">
            <p>
              Feugiat qualisque quo an. Labores officiis te nam.
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
